apps/web/src/app/page.tsx

```diff
@@ -1,17 +1,26 @@
 import AddressMevLookup from "@/components/AddressMevLookup";
+import AddressMevLookupWrapper from "@/components/AddressMevLookupWrapper";
 import WhatIsMevInlineButton from "@/components/Button/WhatIsMevInlineButton";
+import MevExtractionsLiveDashboard from "@/components/MevExtractionsLiveDashboard";
 
 export default function Home() {
   return (
     <div>
-      <h1 className="font-primary text-3xl leading-[30px] text-brand-30 mb-[32px]">
-        Your wallet is bleeding. I can prove it.
-        <br />
-        <span className="text-brand-20">
-          MEV <WhatIsMevInlineButton /> is robbing you on every trade.
-        </span>
-      </h1>
-      <AddressMevLookup />
+      <AddressMevLookupWrapper>
+        <h1 className="font-primary text-3xl leading-[30px] text-brand-30 mb-8">
+          Your wallet is bleeding. I can prove it.
+          <br />
+          <span className="text-brand-20">
+            MEV <WhatIsMevInlineButton /> is robbing you on every trade.
+          </span>
+        </h1>
+
+        <AddressMevLookup />
+      </AddressMevLookupWrapper>
+
+      <div className="pt-[128px] md:pt-[70px] max-md:pt-[40px]">
```

Comment:
This should use built in tailwind variables, please let me know if you need help finding what they are!